osmaxx-py/osmaxx/excerptexport/forms/excerpt_export_form.py

```diff
@@ -200,8 +201,14 @@ def save(self, user):
         extraction_order.extraction_configuration = self._generate_extraction_options()
 
         if self.cleaned_data['form_mode'] == FormModeMixin.MODE_EXISTING:
-            existing_excerpt = Excerpt.objects.get(pk=int(self.cleaned_data['existing_excerpts']))
-            extraction_order.excerpt = existing_excerpt
+            pk = self.cleaned_data['existing_excerpts']
+            is_country = pk.startswith(COUNTRY_ID_PREFIX)
+            if is_country:
```

Comment:
I don't find another use of is_country. So you can merge this and the line before.